Misc/NEWS.d/next/Library/2023-11-17-17-34-30.gh-issue-111965.6iMWn0.rst

```diff
@@ -0,0 +1 @@
+Using critical section to make ``io.TextIOWrapper`` thread safe without GIL.
```

Comment:
As I commented before, NEWS.d is not needed for this task

Reply:
Sorry, just noticed it